dojson/utils.py

```diff
@@ -44,6 +44,21 @@ def wrapper(self, key, value, **kwargs):
     return wrapper
 
 
+def ignore_list_elem(f):
```

Comment:
IMHO this decorator is not needed. You can directly raise in the function implementing a rule.